src/glang/target.py

```diff
@@ -83,7 +82,7 @@ def compute_padding(curr_size: int, align_to: int) -> int:
             # Append member to the struct
             struct_size += member_size
 
-        # The size of any object is always a multiple of the object‘s alignment.
```

Comment:
How did we even sneak the non-asic ‘ in here?

Reply:
I think I just copy-pasted that from the spec, wooops